app/views/layouts/application.html.erb

```diff
@@ -39,8 +39,9 @@
           </ul>
           <ul class="nav navbar-nav navbar-right">
             <% if current_user %>
-              <li class="navbar-brand">Logged in as <%= current_user.username %></li>
-              <li><%= link_to "Logout", logout_path %></li>
+              <li class="logged-as-navbar"><%= "Logged as" %></li>
+              <li class="username-navbar"><%= link_to " #{current_user.username} |"%></li>
```

Comment:
You are right, i havent added the path to our route yet. but will get it done tomorrow. Thanks for posting @JeanJoeris